<?xml version="1.0"?>
<launch>

	<arg name="probs" default="0.8, 0.3"/>
	<arg name="passcode" default="42"/>
	<arg name="num_arms" default="2"/>
	<arg name="room_a" default="1"/>
	<arg name="room_b" default="2"/>

    <node name="bandit_server" pkg="competition2" type="bandit_server.py" output="screen" >
        <param name="probs" value="$(arg probs)"/>
        <param name="passcode" value="$(arg passcode)"/>
        <param name="num_arms" value="$(arg num_arms)"/>
        <param name="room_a" value="$(arg room_a)"/>
        <param name="room_b" value="$(arg room_b)"/>
    </node>

</launch>